elasticsearch.yml

```diff
@@ -1,8 +1,11 @@
+network.host: 0.0.0.0
+
 cluster:
   name: elasticsearch
 
 script:
-  disable_dynamic: false
+  inline: on
+  indexed: on
 
```

Comment:
So does this mean we are using this new settings format also for ES 1.x ? Is it compatible?
See <https://www.elastic.co/guide/en/elasticsearch/reference/2.3/breaking_20_setting_changes.html#migration-script-settings>